doc-gen/src/commands/generate.ts

```diff
@@ -193,9 +193,10 @@ function generateClassDocs(
       ts.isConstructorDeclaration(member) ||
       ts.isMethodDeclaration(member) ||
       (ts.isPropertyDeclaration(member) &&
-        member.initializer &&
-        (ts.isArrowFunction(member.initializer) ||
-          ts.isFunctionExpression(member.initializer)))
+        ((member.initializer &&
+          (ts.isArrowFunction(member.initializer) ||
+            ts.isFunctionExpression(member.initializer))) ||
+          (member.type && ts.isFunctionLike(member.type))))
```

Comment:
this is to handle the `SignerLogger.profiled` class property declarations which are still functions. This allows us to fallback on to the type defintion if it exists so we can still generate docs